packages/ui5-application-writer/test/data.test.ts

```diff
@@ -140,7 +145,8 @@ describe('Setting defaults', () => {
                 localVersion: '1.95.6',
                 minUI5Version: '1.80.0',
                 descriptorVersion: '1.12.1',
-                typesVersion: '1.95.5',
+                typesVersion: '1.95.0',
+                esmTypesVersion: `1.90.0`,
```

Comment:
this is a bit un expected. Why a lower version for esm here?

Reply:
This test is ensuring the passed in values are used rather than relying the minUi5Version to generate them.